expertise/create_dataset.py

```diff
@@ -227,9 +227,10 @@ def retrieve_expertise_helper(self, member, email):
 
             paperhash = openreview.tools.get_paperhash('', n['content']['title'])
 
-            if n['id'] not in self.excluded_ids_by_user[member] and paperhash not in seen_keys:
+            if n.get('original') is not None and n['id'] not in self.excluded_ids_by_user[member] and paperhash not in seen_keys:
```

Comment:
shouldn't be 'is None'? filtered_papers are the ones used for the expertise?

Reply:
you are right! Oh boy...